odoo/service/server.py

```diff
@@ -1248,7 +1248,7 @@ def stop(self):
 server_phoenix = False
 
 def load_server_wide_modules():
-    server_wide_modules = {'base', 'web'} | set(odoo.conf.server_wide_modules)
```

Comment:
You probably should leave it here. It could be another PR to remove `web` from server-wide modules.